## 06/12/2022: CVE-2022-28228: Out-of-bounds reads in YDB servers {#CVE-2022-28228}

_Updated on 08/12/2022_

Notified of vulnerability: Max Arnold <arnold.maxim@yandex.ru>

CVE ID: CVE-2022-28228

Link to CVE: <https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-28228>

### Original report {#original-report}

<https://ydb.tech/docs/en/security-changelog#28-11-2022>

### Summary {#brief-description}

An attacker can make a special request to initiate an error. The error message may contain fragments of data from another cluster. An attacker can also cause a denial of service to the cluster.

### Technologies affected {#technologies-affected}

{{ ydb-full-name }} serverless mode.

### Vulnerable products and versions {#vulnerable-products-and-versions}
### Vendor {#vendor}

The vulnerability affects versions before 22.4.44.

Version 22.4.44 is safe.

### Vendor {#vendor}

Yandex

### Recommendations for vulnerability detection and supporting materials {#recommendations-for-vulnerability-detection}

<https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-28228>

### Safe version of vulnerable product or patch {#safe-version}

The vulnerability is fixed as of version 22.4.44.

### Compensatory measures for {{ yandex-cloud }} users {#compensatory-measures}

All measures have been carried out by the service. No additional actions are required.

### Impact on {{ yandex-cloud }} services {#impact-on-yandex-cloud-services}

The vulnerability only extended to the {{ ydb-full-name }} service in serverless mode. At the moment, the vulnerability is closed: all YDB instances have been updated to a secure version.
